Makefile

```diff
@@ -25,14 +25,16 @@ GO_BUILD_FLAGS :=-tags 'include_gcs include_oss containers_image_openpgp gssapi'
 GO_BUILD_FLAGS_DARWIN :=-tags 'include_gcs include_oss containers_image_openpgp'
 GO_BUILD_FLAGS_WINDOWS :=-tags 'include_gcs include_oss containers_image_openpgp'
 
+OSE_BUILD ?=1
+LOCAL_BUILD ?=1
 OUTPUT_DIR :=_output
 CROSS_BUILD_BINDIR :=$(OUTPUT_DIR)/bin
 RPM_VERSION :=$(shell set -o pipefail && echo '$(SOURCE_GIT_TAG)' | sed -E 's/v([0-9]+\.[0-9]+\.[0-9]+)-.*/\1/')
 RPM_EXTRAFLAGS := \
-	--define 'local_build true' \
+	--define 'local_build $(LOCAL_BUILD)' \
+	--define 'ose_build $(OSE_BUILD)' \
 	--define 'os_git_vars ignore' \
 	--define 'version $(RPM_VERSION)' \
-	--define 'dist .el7' \
```

Comment:
is it safe to remove this override here?

Reply:
The dist tag is not usually hardcoded but determined by the build env. On the current OCP 4.5 base image `%{?dist}` expands to `.el7`, so this wouldn't change until e.g. UBI8 is used to build the rpm.

Reply:
this is for dev builds (e.g.  if you are on fedora) so I don't think it breaks but I don't see how that helps okd

Reply:
It doesn't. The approach has changed and we're currently not doing separate oc rpm builds for OKD.